haiku-apps/commandtimer/additional-files/Makefile

```diff
@@ -0,0 +1,138 @@
+## Haiku Generic Makefile v2.6 ## 
+
+## Fill in this file to specify the project being created, and the referenced
+## Makefile-Engine will do all of the hard work for you. This handles any
+## architecture of Haiku.
+
+# The name of the binary.
+NAME = CommandTimer
+
+# The type of binary, must be one of:
+#	APP:	Application
+#	SHARED:	Shared library or add-on
+#	STATIC:	Static library archive
+#	DRIVER: Kernel driver
+TYPE = APP
+
+# 	If you plan to use localization, specify the application's MIME signature.
+APP_MIME_SIG = 
+
+#	The following lines tell Pe and Eddie where the SRCS, RDEFS, and RSRCS are
+#	so that Pe and Eddie can fill them in for you.
+#%{
+# @src->@ 
+
+#	Specify the source files to use. Full paths or paths relative to the 
+#	Makefile can be included. All files, regardless of directory, will have
+#	their object files created in the common object directory. Note that this
+#	means this Makefile will not work correctly if two source files with the
+#	same name (source.c or source.cpp) are included from different directories.
+#	Also note that spaces in folder names do not work well with this Makefile.
+SRCS = \
+	 CommandTimer.cpp  \
+	 CommandTimerApp.cpp  \
+	 CommandTimerView.cpp  \
+	 CommandTimerWindow.cpp  \
+
+
+#	Specify the resource definition files to use. Full or relative paths can be
+#	used.
+RDEFS = \
+
+
+#	Specify the resource files to use. Full or relative paths can be used.
+#	Both RDEFS and RSRCS can be utilized in the same Makefile.
+RSRCS = \
+	 CommandTimer.rsrc  \
```

Comment:
It would be nice to convert CommandTimer.rsrc into CommandTimer.rdef, as you did for BeOhms, and add it, with the Makefile you wrote, to the upstream repo, as you suggested.
Great work, BTW!